src/components/AddressButton.tsx

```diff
@@ -0,0 +1,27 @@
+import { MapPinIcon } from "@heroicons/react/24/solid";
+import Icon from "./Icon";
+
+export const AddressButton = ({
+  name,
+  mapLink,
+  address,
+}: {
```

Comment:
For future reference, don't de-structure arguments here. Use a variable called `props` and use them directly. For example
```typescript
const SubmitButton = (props: { types... }) => (
    <button someProp={props.value}/>
)
```